### PR TITLE
Add --performance-stats option and use this on some tests

### PR DESCRIPTION
**Pull request by keyboardDrummer**
### Description
Add `--performance-stats` option and use this on the SchorrWaite test

### How has this been tested?
Updated `SchorrWaite.dfy`

<small>By submitting this pull request, I confirm that my contribution is made under the terms of the [MIT license](https://github.com/dafny-lang/dafny/blob/master/LICENSE.txt).</small>

### CODE REVIEW

File: Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/SubsetTypes.dfy [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// RUN: %exits-with 4 %build "%s" --relax-definite-assignment --allow-axioms > "%t"
+// RUN: %exits-with 4 %verify "%s" --performance-stats=10 --relax-definite-assignment --allow-axioms > "%t"
```
Comment (MikaelMayer) [inline Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/SubsetTypes.dfy:1]: It would be worth documenting this attribute some where, wouldn't it?

Comment (keyboardDrummer) [reply Source/IntegrationTests/TestFiles/LitTests/LitTest/dafny0/SubsetTypes.dfy:1]: I meant to make it hidden, as a language developer option, but forgot to do so. I've done that now.

**Review by MikaelMayer — APPROVED** [review]
I would have preferred this option to be documented and available for anyone since it looks useful.

**Comment by keyboardDrummer** [comment]
> I would have preferred this option to be documented and available for anyone since it looks useful.

Can you elaborate on what you would like? It sounds like you want it to be a public option, but I'm not sure what the use-case for Dafny users would be.

**Pull request merged**